helm/cluster-openstack/values.schema.json

```diff
@@ -235,7 +235,6 @@
         "controlPlane",
         "kubernetesVersion",
         "managementCluster",
-        "nodeCIDR",
```

Comment:
Kind of weird to include it with this change. I guess it always worked because we defaulted it to an empty value?

Reply:
Yes. You are right. Setting it with empty string works. I removed this line. Thanks for reviewing my PRs :D